tests/test_structure/test_create_structure.py

```diff
@@ -0,0 +1,32 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.structure.create_structure_fixtures import (
+    structure_mutation_query,
+    structure_mutation_response,
+    empty_field_structure_mutation
+)
+
+
+class TestCreateStructure(BaseTestCase):
+
+    def test_structure_creation(self):
+        """
+        Testing for Structure creation
+        """
+
+        CommonTestCases.admin_token_assert_equal(
+            self,
+            structure_mutation_query,
+            structure_mutation_response
+        )
+
+    def test_structure_creation_when_not_admin(self):
```

Comment:
good work @daniellamarr could you also add docstrings to this test and the one on line `28`

Reply:
Thank you @dnuwa, I will do that.

Reply:
Done